runtime/flangrti/CMakeLists.txt

```diff
@@ -113,6 +113,11 @@ find_library(
   HINTS ${CMAKE_BINARY_DIR}/lib)
 target_link_libraries(flangrti_shared ${LIBPGMATH})
 
+# BSDs need libexecinfo
+if( ${TARGET_OS} STREQUAL "OpenBSD" )
+  target_link_libraries(flangrti_shared ${FLANG_LIBEXECINFO})
```

Comment:
Will flang-driver need to add libexecinfo to the linker command line?

Reply:
No; I do a static link of libexecinfo into flangrti.so
I also set up the flang driver to do all the linking magic behind the scenes. Users simply do something like:
`$ flang myfile.F95`
and everything just works.

See https://github.com/openbsd/ports/blob/master/lang/flang/driver/patches/patch-lib_Driver_ToolChains_OpenBSD_cpp#L54